### PR TITLE
[README] spaceauth - adopt to monogem changes

### PR DESCRIPTION
**Pull request by hjanuschka**
as discussed here: https://github.com/fastlane/fastlane/issues/7562

### CODE REVIEW

File: spaceship/README.md [hunk 1]
```diff
@@ -112,10 +112,10 @@ This requires you to install `pry` using `sudo gem install pry`. `pry` is not in
 
 When your Apple account has 2 factor verification enabled, you'll automatically be asked to verify your identity using your phone. The resulting session will be stored in `~/.spaceship/[email]/cookie`. The session should be valid for about one month, however there is no way to test this without actually waiting for over a month.
 
-Since your CI system probably doesn't allow you to input values (like the verification code), you can use `spaceauth`:
+Since your CI system probably doesn't allow you to input values (like the verification code), you can use `fastlane spaceship spaceauth`:
 
 ```sh
-spaceauth -u apple@krausefx.com
+fastlane spaceship spaceauth -u apple@krausefx.com
```
Comment (KrauseFx) [inline spaceship/README.md]: This should be without `spaceship`, right?

**Comment by hjanuschka** [comment]
@KrauseFx ohhh sorry i missed that - updated

**Review by milch — APPROVED** [review]
Thanks, looks good! 🚀

**Comment by KrauseFx** [comment]
👍

**Pull request merged**